doctr/travis.py

```diff
@@ -150,8 +150,9 @@ def setup_GitHub_push(deploy_repo, auth_type='deploy_key', full_key_path='github
     TRAVIS_BRANCH = os.environ.get("TRAVIS_BRANCH", "")
     TRAVIS_PULL_REQUEST = os.environ.get("TRAVIS_PULL_REQUEST", "")
 
-    if TRAVIS_BRANCH != "master":
-        print("The docs are only pushed to gh-pages from master", file=sys.stderr)
+    if TRAVIS_BRANCH != "master" and no_require_master:
+        print("The docs are only pushed to gh-pages from master. To force pushing from "
```

Comment:
"To *allow* pushing". Force pushing sounds like `git push --force`, which we are definitely not doing.